app/views/students/show.html

```diff
@@ -0,0 +1,125 @@
+<div ng-controller='StudentsShowCtrl as controller'>
+	<!-- Page Heading -->
+	<div class="row">
+		<div class="col-xs-12">
+			<h2 class="page-header">
```

Comment:
I know that I've told you to use `h2` as a page header in #63, but I made a mistake. Sorry for that. Please update this to `h1`